packages/api/src/entity/library_item.ts

```diff
@@ -196,4 +196,28 @@ export class LibraryItem {
 
   @Column('text', { nullable: true })
   recommenderNames?: string[] | null
+
+  @Column('timestamptz')
+  hiddenAt?: Date | null
+
+  @Column('timestamptz')
+  sharedAt?: Date | null
+
+  @Column('text')
+  sharedBy?: string | null
+
+  @Column('jsonb')
+  links?: any | null
+
+  @Column('text')
+  previewContent?: string | null
+
+  @Column('timestamptz')
+  seenAt?: Date | null
+
+  @Column('boolean')
```

Comment:
I think before we were going to do `addedToLibraryAt?: Date` ?

Reply:
Yeah, I will add it